### PR TITLE
Examine page header

### PR DESCRIPTION
**Pull request by jaclynperrone**
Would love some feedback for this one! This is the first pass at redesigning the header on the examine page. This commit does a few things:

- Move CQL filter, VCF comparison and # of records into the table header
- Only the VCF table horizontally scrolls, instead of the entire page
- `RecordsShown` is now its own component (instead of being part of `StatsSummary`). This is to give greater flexibility to the layout.
- Updates pdiff screenshots

Thanks!

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/cycledash/774)

### CODE REVIEW

**Comment by jaclynperrone** [comment]
Pulling in @danvk as well for some feedback!

**Comment by danvk** [comment]
There are some clipping issues with the screenshots, but I think the redesign looks great!

**Comment by ihodes** [comment]
Looks great, big improvement! I wouldn't mind the stats taking up less room somehow—they get a ton of space. Maybe file an issue to do this if you have plans to change it more?

File: tests/pdifftests/test.py [hunk 1]
```diff
@@ -3,10 +3,8 @@
 from selenium.webdriver.support.ui import Select
 from selenium.webdriver.common.action_chains import ActionChains
```
Comment (ihodes) [inline tests/pdifftests/test.py]: This is a massive silly nit, so don't worry about changing it, but in general 2 newlines between imports and top-level definitions.

Comment (jaclynperrone) [inline tests/pdifftests/test.py]: Ahhh good to know! Thought I was doing a little light cleaning :)

**Comment by jaclynperrone** [comment]
@danvk @ihodes Thanks for taking a look!!

@ihodes Those stats have been tricky, that's for sure. It makes sense that they live where they live, and I agree that they take up a lot of room. I am going to reduce the padding a little bit to tighten them up. And while I'm at it, I'll fix that `test.py` file as well :)

I think that any further changes to those stats will be handled in another PR. I would love sit together and figure out a friendlier, more straightforward way of presenting that information. I remember we played around with the idea of having a legend that says what the symbols mean after the comparison is done.

**Comment by jaclynperrone** [comment]
Closes #710

**Pull request merged**